contributors/design-proposals/job.md

```diff
@@ -79,8 +105,21 @@ type JobSpec struct {
     // job should be run with. Defaults to 1.
     Completions *int
 
+    // Optional duration in seconds relative to the startTime that the job may be active
+    // before the system tries to terminate it; value must be positive integer
+    ActiveDeadlineSeconds *int
+
+    // Optional number of retries, before marking this job failed.
+    BackoffLimit *int
+
+    // Optional time (in seconds), how log a job should be retried before marking it failed.
```

Comment:
- [ ] s/), how/) specifying how/
- [ ] Did you consider using type `*uint` instead?
  That could encourage unintentional overflow when used with [`time`](https://golang.org/pkg/time/).